engine/table/src/main/java/io/deephaven/engine/util/AbstractScriptSession.java

```diff
@@ -146,8 +146,10 @@ public synchronized final Changes evaluateScript(final String script, final @Nul
                 final SafeCloseable ignored = LivenessScopeStack.open(this, false)) {
 
             try {
-                // actually evaluate the script
-                executionContext.apply(() -> evaluate(script, scriptName));
+                // Actually evaluate the script; use the enclosing auth context, since AbstractScriptSession's
+                // ExecutionContext never has a non-null AuthContext
```

Comment:
double negative.

Reply:
I'll update this in a subsequent PR, unless something goes wrong with unit testing.